### PR TITLE
Validation display fix

### PR DESCRIPTION
**Pull request by tom-price**
Addresses issues #153 and #349. PR #308 also addresses part of what this PR addresses; however, it took me a few goes reading the changes to understand the changes as written.

[Here's a JS Fiddle](https://jsfiddle.net/Bengt/tvow8s6x/21/
) of what this PR fixes. The first section is with the added divs and the 2nd shows what it's like without them present.

@dyve, sorry for the resubmit of #350 but I renamed the branch in my fork while working on another issue and it closed the old PR.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 22.2 (2022-09-02)
```
Comment (dyve) [inline CHANGELOG.md:3]:
```suggestion
## 22.2 (in development)
```

File: tests/test_bootstrap_field_input_text.py [hunk 2]
```diff
@@ -8,6 +8,10 @@ class CharFieldTestForm(forms.Form):
     test = forms.CharField()
 
 
+class CharFieldRequiredTestForm(forms.Form):
+    test = forms.CharField(min_length=1)
```
Comment (dyve) [inline tests/test_bootstrap_field_input_text.py:12]: This does not perform an assert. Is that intentional?

Comment (tom-price) [reply tests/test_bootstrap_field_input_text.py:12]: This test form was created to produce a form that will fail a `form.is_valid()` check. The response can then be rendered to check the correct HTML is returned for the failing form field. It's used [here](https://github.com/tom-price/django-bootstrap5/blob/9c1ee5b59dd6b7f42eceaa941392b324ed1f9329/tests/test_bootstrap_field_input_text.py#L155-L173) to do so.

**Review by dyve — APPROVED** [review]

**Comment by dyve** [comment]
Failing packaging check, let's see if we can fix that.

**Pull request merged**